cli/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+
+## 12.14.1
+
+_Released 06/20/2023 (PENDING)_
+
+**Bugfixes:**
+
+- The [`videoCompression`](https://docs.cypress.io/guides/references/configuration#Videos) configuration option now accepts both a boolean or a Constant Rate Factor (CRF) number between `0` and `51`. The `videoCompression` default value is still `32` CRF and when `videoCompression` is set to `true` the default of `32` CRF will be used. Addresses [#26658](https://github.com/cypress-io/cypress/issues/26658).
```

Comment:
```suggestion
- The [`videoCompression`](https://docs.cypress.io/guides/references/configuration#Videos) configuration option now accepts both a boolean or a Constant Rate Factor (CRF) number between `1` and `51`. The `videoCompression` default value is still `32` CRF and when `videoCompression` is set to `true` the default of `32` CRF will be used. Addresses [#26658](https://github.com/cypress-io/cypress/issues/26658).
```

Reply:
0 is still a valid number for CRF, we just don't compress the video since a CRF of 0 is essentially a non compressed video

Reply:
I think its better to be explicit that this disabled compression even if it is a valid CRF value. If I didn't have context, I'd expect 0 to compress since false is called out to prevent compression.

Reply:
I think I got this added in 53a07c986d